src/Propel/Runtime/Collection/Collection.php

```diff
@@ -594,8 +595,9 @@ public function __call($name, $params)
             $format = substr($name, 2);
             $usePrefix = isset($params[0]) ? $params[0] : false;
             $includeLazyLoadColumns = isset($params[1]) ? $params[1] : true;
+            $keyType = isset($params[2]) ? $params[2] : TableMap::TYPE_PHPNAME;
```

Comment:
Same here.

Reply:
Changed it, and the lines above as well. Thank you!